scripts/storage-emulator-integration/tests.ts

```diff
@@ -244,6 +246,19 @@ describe("Storage emulator", () => {
         });
       });
 
+      describe("#delete()", () => {
+        it.only("should properly delete a file from the bucket", async () => {
+          await testBucket.upload(toDeleteFilePath);
+          await testBucket.file(toDeleteFilePath.split("/").slice(-1)[0]).delete();
+
+          try {
+            fs.statSync(toDeleteFilePath);
+          } catch (err) {
```

Comment:
This also passes if file exists. To mitigate this just assert file doesn't exist (!fs.existsSync).

FYI, in general testing for errors should use the formula `
expect(func).to.throw().with.property('code', 'ENOENT');`

Reply:
Sure, I'll use this check instead 👍

Reply:
I also see that I forgot to remove the `it.only` on the test.
I'll remove that at the same time.